plugins/org.python.pydev.core/helpers/load-conda-vars.bat

```diff
@@ -14,11 +14,10 @@
 @set "CONDA_PATH_BACKUP=%PATH%"
 @set "PATH=%CONDA_PREFIX%;%CONDA_PREFIX%\Scripts;%CONDA_PREFIX%\Library\bin;%PATH%"
 
-@REM Run any activate scripts
-@if exist "%CONDA_PREFIX%\etc\conda\activate.d" (
-    @pushd "%CONDA_PREFIX%\etc\conda\activate.d"
-    @for %%g in (*.bat) do @call "%%g"
-    @popd
+@REM activate conda and get the environment
+@if exist "%CONDA_PREFIX%" (
+	@call do conda activate %CONDA_PREFIX%
+	@call do conda list
```

Comment:
Why is `conda list` being called here?

Reply:
At first I thought conda list was what output the environment variables... I forgot to remove it. My bad